pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -773,6 +773,16 @@ func (c *Cluster) getActivateOSDInitContainer(configDir, namespace, osdID string
 	return volume, container
 }
 
+func getBlockDevMapperContext() *v1.SecurityContext {
+	return &v1.SecurityContext{
+		Capabilities: &v1.Capabilities{
+			Add: []v1.Capability{
+				"MKNOD",
```

Comment:
Please add a small comment on why we need `MKNOD` even though it might sound obvious due to the block copy. Thanks

Reply:
Done. Function accompanied with the following comment:

```
// The blockdevmapper container copies the device node file, which is regarded as a device special file.
// To be able to perform this action, the CAP_MKNOD capability is required.
// Provide a securityContext which requests the MKNOD capability for the container to function properly.
```